spidermon/contrib/scrapy/monitors.py

```diff
@@ -44,6 +44,38 @@ def test_minimum_number_of_items(self):
         self.assertTrue(item_extracted >= self.minimum_threshold, msg=msg)
 
 
+@monitors.name("Extracted Expected Items Monitor")
+class ExpectedItemCountMonitor(BaseScrapyMonitor):
+    """Check if spider extracted the expected minimum number of items provided
+    in a stats `spidermon/expected_item_scraped_count`.
+
+    You can configure it using ``SPIDERMON_EXPECTED_ITEMS_THRESHOLD`` setting.
```

Comment:
Looking for a better name for this setting also.
Moreover, not sure it should a float [0, 1] or an in [0, 100].

One example to think here Say that I allow 10% of missed items.
Is it better:
* `SPIDERMON_EXPECTED_ITEMS_THRESHOLD = 0.1`
* `SPIDERMON_EXPECTED_ITEMS_THRESHOLD = 0.9`

Reply:
I think more obvious would be `1` if we expect all `expected_item_scraped_count` and `0` if we expect 0.
So `SPIDERMON_EXPECTED_ITEMS_THRESHOLD = 0.9`